projects/packages/my-jetpack/_inc/components/my-jetpack-tab-panel/overview/content.tsx

```diff
@@ -0,0 +1,31 @@
+import { Col, Container } from '@automattic/jetpack-components';
+import { isAtomicSite } from '@automattic/jetpack-script-data';
+import ConnectionsSection from '../../connections-section';
+import PlansSection from '../../plans-section';
+import ProductCardsSection from '../../product-cards-section';
+import styles from './styles.module.scss';
+
+/**
+ * The Overview content component.
+ *
+ * @return The rendered component.
+ */
+export function OverviewContent() {
+	return (
+		<div>
+			<div className={ styles.products }>
+				<ProductCardsSection />
+			</div>
+			<div className={ styles.footer }>
+				<Container horizontalSpacing={ 0 } className={ styles[ 'footer-container' ] }>
+					<Col sm={ 4 } md={ 4 } lg={ 6 }>
+						<PlansSection />
+					</Col>
+					<Col sm={ 4 } md={ 4 } lg={ 6 }>
+						{ ! isAtomicSite() && <ConnectionsSection /> }
```

Comment:
I suggest assigning the function to a variable and using the variable in the return block.

Reply:
Reason?